library/core/src/panic/location.rs

```diff
@@ -32,7 +34,16 @@ use crate::fmt;
 #[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 #[stable(feature = "panic_hooks", since = "1.10.0")]
 pub struct Location<'a> {
+    #[cfg(bootstrap)]
```

Comment:
All the `#[cfg(bootstrap)]` and `#[cfg(not(bootstrap))]` parts should no longer be necessary as of https://github.com/rust-lang/rust/pull/119899